package.json

```diff
@@ -121,6 +121,8 @@
     "@types/jest": "24.9.1",
     "@types/jest-axe": "3.2.2",
     "@types/jju": "1.4.1",
+    "@types/json-schema": "^7.0.8",
```

Comment:
Development dependency, typings are used in `packages/babel-make-styles/src/schema.ts`.